mtgjson5/resources/sealed_name_fixes.json

```diff
@@ -0,0 +1,12 @@
+{
+    "DotP": "Duels of the Planeswalkers",
+    "M15 Prerelease": "Magic 2015 Prerelease",
+    "All 5": "Set of 5",
+    "M10 ": "",
```

Comment:
Is it intentional the M1X have a space after the name

Reply:
yes, it's needed to remove an extra space

Reply:
if you want i can just remove the tag and then add an extra entry to replace "  " with " "